change/@fluentui-react-aria-1b519f98-87ae-4fe2-be29-f46d1fe6fec4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32896&commitId=43b486ba6404b26a162a86de7d1346c3e56fa09c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|18| Changed|
|Avatar Converged.badgeMask.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|996| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>